### DISCUSSION TITLE
Qt is a bad choice if the SourceMonitor is to be Windows only - choose C#/WPF instead

### DISCUSSION THREAD

**Discussion by ren333** [SourceMonitor/SourceMonitorOS#22] [Ideas]
C++ classes can be used in C#. UI is also easier to develop via C#/WPF.
If you want to support Qt, so be it, but we need to get this project out the door as fast as possible, since people are depending on it.

I'm still waiting for Qt to download! 36Gb! After 6 hours of waiting, there's still 6 hours to go, and I still need to sort out CMake integration! Not ideal.

**Comment by Wolf-SO** [comment]
Should a possible replacement of Qt by wxWidgets be investigated too?
